src/main/java/com/luxoft/svinenkov/movieland/controller/MovieController.java

```diff
@@ -0,0 +1,73 @@
+package com.luxoft.svinenkov.movieland.controller;
+
+import com.luxoft.svinenkov.movieland.entity.Movie;
+import com.luxoft.svinenkov.movieland.service.MovieService;
+import com.luxoft.svinenkov.movieland.utils.JsonJacksonConverter;
+import com.luxoft.svinenkov.movieland.utils.JsonManualConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+@Controller
+@RequestMapping(path = "/v1/movie", produces = "application/json; charset=utf-8")
```

Comment:
move "/v1" into web.xml - to have ability connect different dispatcher

Reply:
done